test/__snapshots__/source-map-options.test.js.snap

```diff
@@ -0,0 +1,551 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`source map and parallel: asset main.0c220ec66316af2c1b24.js 1`] = `
+Object {
+  "map": Object {
+    "file": "x",
+    "mappings": "gCAKAA,EAAAC,QAAA,WAEAC,QAAAC,IAAAC",
+    "names": Array [
+      "module",
+      "exports",
+      "console",
+      "log",
+      "b",
+    ],
+    "sources": Array [
+      "/home/evilebottnawi/IdeaProjects/uglifyjs-webpack-plugin/test/fixtures/entry.js",
+    ],
```

Comment:
This will always fail somewhere else with an absolute file path

Reply:
@d3viant0ne fixed